Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -66,6 +67,224 @@ constexpr int MACH_O_POWERPC_750 = 9;
 
 const s64 GDB_UPDATE_CYCLES = 100000;
 
+const char* QUERY_XFER_TARGET = "qXfer:features:read:target.xml:";
+const char* QUERY_XFER_MEMORY_MAP = "qXfer:memory-map:read::";
+
+const char* TARGET_MEMORY_MAP_XML_NO_MMU =
+    "<memory-map version=\"1.0\">"
+    "<memory type=\"ram\" start=\"0x7E000000\" length=\"0x2000000\"/>"
+    "<memory type=\"ram\" start=\"0x80000000\" length=\"0x1800000\"/>"
+    "<memory type=\"ram\" start=\"0x90000000\" length=\"0x4000000\"/>"
+    "</memory-map>";
```

Comment:
I think this is a good use for [raw string literals](https://en.cppreference.com/w/cpp/language/string_literal):

```C++
const char* TARGET_MEMORY_MAP_XML_NO_MMU =
    R"(<memory-map version="1.0">
<memory type="ram" start="0x7E000000" length="0x2000000"/>
<memory type="ram" start="0x80000000" length="0x1800000"/>
<memory type="ram" start="0x90000000" length="0x4000000"/>
</memory-map>)";
```

Note that this is not 100% equivalent, as raw string literals also include the newlines, but I think that's fine.